NMC Workshop Presentation

cristinamullin opened the pull request:
Collaborators - please review and add content directly to this NMC-Workshop-Draft branch through commits. Let's use this space to share feedback as comments or code update suggestions on the open pull request.

mhweber commented:
@cristinamullin you probably saw I added a few slides and will continue to make commits on that section next few days - thanks!

hillarymarler commented:
Made a few small suggested changes, let's chat about how I can help with filling in the CWA examples. I want to make sure I understand what your are envisioning for that section.

cristinamullin commented:
> @cristinamullin you probably saw I added a few slides and will continue to make commits on that section next few days - thanks!

@mhweber @michaeldumelle I just merged your edits into this new branch [NMC-Workshop-Draft](https://github.com/USEPA/EPATADA/tree/NMC-Workshop-Draft). Please work from here now. I converted this to a .Rmd presentation because the TADA pkgdown site is not compatible with quarto presentations.

mhweber commented:
Just pushed StreamCat content - I'm in the process of updating StreamCatTools and hope to flesh out section where I left a note in .Rmd after I update the package
